crates/loader/Cargo.toml

```diff
@@ -27,6 +27,7 @@ shellexpand = "3.1"
 spin-locked-app = { path = "../locked-app" }
 spin-common = { path = "../common" }
 spin-manifest = { path = "../manifest" }
+spin-variables = { path = "../variables" }
```

Comment:
This loader dependency appears to be a new dependency rather than a rename.  Have we really introduced this dependency?  There seem to be no associated code changes so I'm not sure why it would be needed.

Reply:
Oh! Good catch, must have missed this when rebasing onto the v2 manifest.